src/ge-shared.js

```diff
@@ -13,16 +26,24 @@ export function parseCSSList (value) {
 
 const COLOR_PARSE_REGEX = /((^[\w]+$)|(^\#[a-f\d]{3,6}$))/i;
 
+/**
+ * Parse simple color description.
+ *
+ * @param  {[type]} colorString  [description]
+ * @param  {[type]} def          [description]
+ * @return {[type]}              [description]
+ */
 export function parseSimpleColor (colorString, def = undefined) {
   return COLOR_PARSE_REGEX.test(colorString) ? colorString : def;
 }
 
 /**
+ * Geo extend element description.
  *
- * @param {string} elementName
- * @param {HTMLElement} nativeElement
- * @param {object} options
- * @returns {HTMLElement}
+ * @param  {string}      elementName    [description]
+ * @param  {HTMLElement} nativeElement  [description]
+ * @param  {object}      options        [description]
+ * @return {HTMLElement}                [description]
```

Comment:
The spacing here is auto formatted with `npm run fix`. But is also fully customizable with `check-line-alignment` rule.